desktop/libs/indexer/src/indexer/api3_tests.py

```diff
@@ -104,3 +104,22 @@ def test_col_names():
   assert_true('date_1_' in columns_name)
   assert_true('hour_1' in columns_name)
   assert_true('minute' in columns_name)
+
+
+def test_guess_format_excel_remote_file():
+  with patch('indexer.api3.pd') as pd:
+    with patch('indexer.api3.MorphlineIndexer') as MorphlineIndexer:
+      pd.return_value = Mock()
+      MorphlineIndexer.return_value = MagicMock()
```

Comment:
Do we really need this?
IIRC, since we are already patching above them above, it by default has Mock object unless we want `return_value` to be something specific.

Reply:
Nice catch! Thanks